### PR TITLE
Fixing aggregations in the API client

### PR DESCRIPTION
**Pull request by kiddinn**
Removing the need to depend on the aggregation manager, and thus removing dependencies on timesketch library in the API client.

Also adding support for storing and listing stored aggregations.

### CODE REVIEW

File: api_client/python/timesketch_api_client/client.py [hunk 1]
```diff
@@ -688,6 +709,43 @@ def run_aggregator(
...
+        }
+
+        response = self.api.session.post(resource_url, json=form_data)
+        response_dict = response.json()
+
+        objects = response_dict.get('objects', [])
+        if not objects:
+          return
+
+        _ = self.lazyload_data(refresh_cache=True)
```
Comment (berggren) [inline api_client/python/timesketch_api_client/client.py]: no need to declare a variable here

File: api_client/python/timesketch_api_client/client.py [hunk 2]
```diff
@@ -874,32 +932,23 @@ def table(self):
...
+            self.name, self.parameters, as_pandas=False)
+
+        meta = data.get('meta', {})
+        vega_spec = meta.get('vega_spec')
+
+        if not vega_spec:
+            return altair.Chart(pandas.DataFrame()).mark_point()
 
-        return chart_obj.generate()
+        vega_string = json.dumps(vega_spec)
```
Comment (berggren) [inline api_client/python/timesketch_api_client/client.py]: s/vega_string/vega_spec_string/

File: api_client/python/timesketch_api_client/client.py (hunk 1 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/client.py]: ignored after offline chat

File: api_client/python/timesketch_api_client/client.py (hunk 2 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/client.py]: ack

**Pull request merged**